src/scitiff/_resources/metadata-schema.json

```diff
@@ -0,0 +1,383 @@
+{
+  "$defs": {
+    "ImageDataArrayMetadata": {
+      "description": "Image DataArray Metadata without values(image).",
+      "properties": {
+        "data": {
+          "$ref": "#/$defs/ImageVariableMetadata"
+        },
+        "masks": {
+          "additionalProperties": {
+            "$ref": "#/$defs/ScippVariable"
+          },
+          "title": "Masks",
+          "type": "object"
+        },
+        "coords": {
+          "additionalProperties": {
+            "$ref": "#/$defs/ScippVariable"
+          },
+          "title": "Coords",
+          "type": "object"
+        },
+        "name": {
+          "anyOf": [
+            {
+              "type": "string"
+            },
+            {
+              "type": "null"
+            }
+          ],
+          "default": null,
+          "title": "Name"
+        }
+      },
+      "required": [
+        "data"
+      ],
+      "title": "ImageDataArrayMetadata",
+      "type": "object"
+    },
+    "ImageVariableMetadata": {
```

Comment:
This does not contain values. But it is used above as the data of `"ImageDataArrayMetadata"`.

Reply:
Because it shouldn't have data itself in here.